src/genie/libs/parser/iosxe/tests/ShowIpSlaSummary/cli/equal/golden_output_1_expected.py

```diff
@@ -0,0 +1,39 @@
+expected_output = {
+    'probes': {
+            '1': {'destination': '123.23.213.32',
```

Comment:
Please add `codes` with what we have top of the line which is `*`.
```suggestion
            '1': {'codes': 'active',
```
(convert the `*` based on explanatory note)
```
Codes: * active, ^ inactive, ~ pending
```
corresponding output:
```
*1           tcp-connect 123.23.213.32     RTT=44      OK          21 seconds ago 
```

Reply:
I have used probe_status instead of codes as I thought it to be more explanatory.

Reply:
ok. thanks for update.